backend/layers/processing/schema_migration.py

```diff
@@ -24,88 +26,93 @@ class SchemaMigrate:
     def __init__(self, business_logic: BusinessLogic):
         self.business_logic = business_logic
 
-    def gather_collections(self) -> Dict[str, Dict[str, List[Dict[str, str]]]]:
+    def gather_collections(self) -> List[Dict[str, str]]:
         """
         This function is used to gather all the collections and their datasets that will be migrated
         :return: A dictionary with the following structure:
-        {
-            "published": {
-                "<collection_id>":
-                    [
-                        {"dataset_id": "<dataset_id>", "dataset_version_id": "<dataset_version_id>"},
-                        {...},
-                         ...
-                     ],
-                ...
-            },
-            "revision": {
-                "<collection_version_id>":
-                    [
-                        {"dataset_id": "<dataset_id>", "dataset_version_id": "<dataset_version_id>"},
-                        {...},
-                         ...
-                     ],
-                ...
-            },
-            "private": {
-                "<collection_id>":
-                    [
-                        {"dataset_id": "<dataset_id>", "dataset_version_id": "<dataset_version_id>"},
-                        {...},
-                         ...
-                     ],
-                ...
-            },
-        }
+        [
+            {"can_open_revision": True, "collection_id": "<collection_id>"},
+            {"can_open_revision": False, "collection_id": "<collection_id>", "collection_version_id":
+            "<collection_version_id>"},
+            {"can_open_revision": False, "collection_id": "<collection_id>"},
+            ...
+        ]
         """
 
         published_collections = self.business_logic.get_collections(CollectionQueryFilter(is_published=True))
         unpublished_collections = self.business_logic.get_collections(CollectionQueryFilter(is_published=False))
 
-        response = {"published": {}, "private": {}, "revision": {}}
-
-        def get_datasets(_version):
-            datasets = []
-            for dataset in _version.datasets:
-                datasets.append({"dataset_id": dataset.dataset_id.id, "dataset_version_id": dataset.version_id.id})
-            return datasets
+        response = []
 
         collections = itertools.chain(unpublished_collections, published_collections)
         # evaluate unpublished collections first, so that published versions are skipped if there is an active revision
         has_revision = []  # list of collections to skip if published with an active revision
         for collection in collections:
+
+            # TODO <testing code>
```

Comment:
This snippet was introduced to test in rdev. This is because pre-seeded collections can't be migrated (they don't belong to the environment). It should be removed before merging.